<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Книга рецептов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="sort">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>145</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>145</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="marks">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>145</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Возможности</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_3"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Добавить рецепт</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Удалить рецепт</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Редактировать рецепт</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
